microsoft-edge/progressive-web-apps-chromium/microsoft-store.md

```diff
@@ -95,18 +95,19 @@ Now that you have your app publishing information, generate a Windows app packag
 To generate an app package, complete the following actions.  
 
 1.  Navigate to [PWA Builder][PwabuilderMain].  
-1.  Type the URL of your PWA.  
-1.  Choose **Start** > **Build My PWA** > **Windows** > **Options**.  
+1.  Type the URL of your PWA and click **Start**.  
+1.  Once your report is complete, make sure your PWA is store-ready. If your PWA score is too low, you can visit the **Manifest Options** and **Service Worker Options** and check out the sections that need work.
+1.  When your PWA is ready to be published, click **Next** and select the **Store Package** button in the Windows section of the publish page.
 1.  Paste the following values that you saved in the [Create an app reservation](#create-an-app-reservation) section.  
     *   **Package ID**  
     *   **Publisher ID**  
     *   **Publisher Display Name**  
         
-    :::image type="complex" source="./media/pwabuilder-publisher-info.msft.png" alt-text="Paste publisher information into PWABuilder" lightbox="./media/pwabuilder-publisher-info.msft.png":::
+    :::image type="complex" source="./media/pwabuilder-windows-package-options.png" alt-text="Paste publisher information into PWABuilder" lightbox="./media/pwabuilder-windows-package-options.png":::
        Paste publisher information into PWABuilder  
     :::image-end:::  
     
-1.  Choose **Done**.  
+1.  Choose **Generate**.  
 1.  To download your Windows app package, choose **Download**.
```

Comment:
```suggestion
1.  To download your Windows app package, select **Download**.
```
I don't plan a complete Edit pass of this .md file, b/c this PR is in a fork so I can't do a local edit on the branch and then push my commits.